go.mod

```diff
@@ -29,7 +29,9 @@ replace (
 )
 
 require (
+	github.com/google/uuid v1.6.0
 	github.com/harvester/harvester v1.3.2
+	github.com/harvester/harvester-load-balancer v0.3.0
 	github.com/harvester/harvester-network-controller v0.5.4
```

Comment:
As LB v0.3.0 had some known bugs, could we bump to: Harvester v1.4.0 and LB v.0.4.3?

Reply:
The latest iteration of this PR includes a bump to Harvester v1.5.0 and LB v1.5.0